app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")

```

Comment:
Consider using a context manager (`with` statement) for the SQLite connection to ensure that it is properly closed after operations are completed. This will help prevent potential resource leaks.